nextflow.config

```diff
@@ -0,0 +1,23 @@
+
+profiles {
+  
+  module {
```

Comment:
And if we call this `standard` instead of `module`, then it will be used by default if `-profile` is not used?